CHANGELOG.md

```diff
@@ -2,35 +2,107 @@
+      - `TransactionEventInput`:
+        - `pspReference` - The PSP Reference of the transaction.
+        - `message` - Message related to the transaction's event.
+
+    - Deprecate webhooks:
+      - `TRANSACTION_ACTION_REQUEST` - Use `TRANSACTION_CHARGE_REQUESTED`, `TRANSACTION_REFUND_REQUESTED`, `TRANSACTION_CANCELATION_REQUESTED` instead.
+    - Deprecate object fields:
+      - `TransactionItem`:
+        - `voidedAmount` - Use `canceledAmount`. This field will be removed in Saleor 3.13 (Preview feature).
+        - `status` - The amounts can be used to define the current status of transactions. This field will be removed in Saleor 3.13 (Preview feature).
+        - `reference` - Use `pspReference` instead. This field will be removed in Saleor 3.13 (Preview feature).
+      - `TransactionEvent`:
+        - `status` - Use `type` instead. This field will be removed in Saleor 3.13 (Preview feature).
+        - `reference` - Use `pspReference` instead. This field will be removed in Saleor 3.13 (Preview feature).
+        - `name` - Use `message` instead. This field will be removed in Saleor 3.13 (Preview feature).
+      - `TransactionActionEnum`
+        - `VOID` - Use `CANCEL` instead. This field will be removed in Saleor 3.13 (Preview feature).
+      - `Order`:
+        - `totalCaptured` - Use `totalCharged` instead. Will be removed in Saleor 4.0
+      - `OrderEvent`:
+        - `status` - Use `TransactionEvent` to track the status of `TransactionItem`. This field will be removed in Saleor 3.13 (Preview feature).
+      - `OrderEventsEnum`:
+        - `TRANSACTION_CAPTURE_REQUESTED` - Use `TRANSACTION_CHARGE_REQUESTED` instead. This field will be removed in Saleor 3.13 (Preview feature).
+        - `TRANSACTION_VOID_REQUESTED` - Use `TRANSACTION_CANCEL_REQUESTED` instead. This field will be removed in Saleor 3.13 (Preview feature).
+
+    - Deprecate input fields:
+      - `TransactionCreateInput` & `TransactionUpdateInput`:
+        - `status` - The amounts can be used to define the current status of transactions. This input field will be removed in Saleor 3.13 (Preview feature).
+        - `type` - Use `name` and `message` instead. This input field will be removed in Saleor 3.13 (Preview feature).
+        - `reference` - Use `pspReference` instead. This input field will be removed in Saleor 3.13 (Preview feature).
+        - `amountVoided` - Use `amountCanceled` instead. This input field will be removed in Saleor 3.13 (Preview feature).
+      - `TransactionEventInput`:
+        - `status` - Status will be calculated by Saleor. This input field will be removed in Saleor 3.13 (Preview feature).
+        - `reference` - Use `pspReference` instead. This input field will be removed in Saleor 3.13 (Preview feature).
+        - `name` - Use `message` instead. This field will be removed in Saleor 3.13 (Preview feature).
+  - Feature preview breaking change:
```

Comment:
Shouldn't this go to "Breaking Changes" section?

Reply:
@maarcingebala It depends if `breaking change` should also include a feature preview notes. I wasn't sure, but I can move it there